usaspending_api/api_contracts/contracts/v2/disaster/overview.md

```diff
@@ -1,12 +1,17 @@
 FORMAT: 1A
 HOST: https://api.usaspending.gov
 
-# Disaster Spending Overview [/api/v2/disaster/overview/]
+# Disaster Spending Overview [/api/v2/disaster/overview/{?def_codes}]
 
 This endpoint provides funding and spending details from emergency/disaster supplemental funding legislation.
 
 ## GET
 
++ Request (application/json)
+    + Parameters
+        + `def_codes` (optional, string) 
```

Comment:
I know it is a pain. Can you copy-paste the DEFC enum from another contract into this one?

Reply:
Actually, I can't. As a query param, this IS considered a string to the computer, since array parsing is up to implementation: https://medium.com/raml-api/arrays-in-query-params-33189628fa68

Reply:
Dredd doesn't complain when it is defined as an array of enum strings:
```
        + `def_codes` (optional, array[DEFC])
```

I'm unsure how that will cause issues since we're documenting the valid values not actually writing the implementation here.